knpy/braid.py

```diff
@@ -197,22 +197,24 @@ def braid_relation2(self,index,inplace=True):
             raise IllegalTransformationException
 
     #Markov moves
-    def conjugation(self,index,inplace = True):
+    def conjugation(self,index1,index2):
         """
-        Conjugates the braid with sigma indexed by index
-        index: int in within the interval [-(n-1),n-1], not equal to zero
+        Conjugates the braid with sigma indexed by index1, inserts a index sigma indexed by index1 and -index1 to the index index2
```

Comment:
I suggest making this clearer than "index sigma indexed by index1 and -index1 to the index index2", maybe rename the index parameter to value or sigma index.